### PR TITLE
Issue 2018/02

### PR DESCRIPTION
**Pull request by garbas**
# Please write a comment what you wish to be included in next newsletter!

This newsletter is going to be published on

    Thursday, 2018-02-01,

all the comments until end of

    Tuesday, 2018-01-30,

will be included.

### [Preview](http://weekly.nixos.org/preview) for the inpatient.
  
---
edit: dates

### CODE REVIEW

File: content/2018/02-life-before-death.rst [hunk 1]
```diff
@@ -122,10 +122,15 @@ Tooling
...
 
+- `Stackage nixpkgs overlays`_
+
+  The folks at Typeable have published a nixpkgs overlay which adds
+  ``pkgs.haskell.packages.stackage`` to the package set.
 
 .. _`nixos/nixos-hardware`: https://github.com/NixOS/nixos-hardware
 .. _`@yegortimoshenko`: https://github.com/yegortimoshenko
 .. _`nixos-mailserver`: https://github.com/r-raymond/nixos-mailserver
+.. _`Stackage nixpkgs overlays`: https://blog.typeable.io/posts/2018-01-19-stackage-overlay.html
```
Comment (garbas) [inline content/2018/02-life-before-death.rst]: i already add this under the news. not sure which section would be better. your call.

Comment (badmutex) [reply content/2018/02-life-before-death.rst]: woops! fixed

**Comment by zimbatm** [comment]
We are looking for mentors for the Google Summer of Code project! Send your PRs here: https://github.com/nix-community/google-summer-of-code

**Comment by mogorman** [comment]
I saw this tweeted out today https://qfpl.io/posts/installing-nixos/ thought it might make a good adition

**Comment by FRidh** [comment]
Pure evaluation mode in Nix Unstable: https://github.com/NixOS/nix/commit/d4dcffd64349bb52ad5f1b184bee5cc7c2be73b4

**Comment by zimbatm** [comment]
It's not really a news but Eelco's slides on Nix 1.12 from NixCon are great! https://schedule.nixcon2017.org//system/event_attachments/attachments/000/000/004/original/talk.pdf?1509289391

**Comment by garbas** [comment]
Probably we should also - somehow - mention that Nix 2.0 is coming as per -> https://github.com/NixOS/nix/issues/1806#issuecomment-362062346

**Comment by badmutex** [comment]
Exciting! I'll add that and plan to merge later today.
@garbas We may need to coordinate sending out the email. How would you like
to do that?

**Comment by badmutex** [comment]
Also marking here to add later
http://sandervanderburg.blogspot.com/2018/01/diagnosing-problems-and-running.html

**Review by badmutex — APPROVED** [review]

**Comment by garbas** [comment]
@badi i forgot to give you access to mailchimp to send it also send it via email out. can you ping me on IRC?

aha also looks like travis didn't deploy as expected. I'm looking into it and then also send the emails.

**Comment by garbas** [comment]
fixed the "merge to production" bug in https://github.com/NixOS/nixos-weekly/commit/9259deb4d9b4ae5baec3c5039ce2e4bc9e29a6a7

**Comment by garbas** [comment]
emails sent. @badi lets catch up online and get you access to mailchimp

**Pull request merged**